reV/rep_profiles/rep_profiles.py

```diff
@@ -599,7 +616,8 @@ def __init__(self, gen_fpath, rev_summary, reg_cols=None,
 
         self._check_req_cols(self._rev_summary, self._reg_cols)
         self._check_req_cols(self._rev_summary, self._weight)
-        self._check_req_cols(self._rev_summary, self._gid_col)
+        self._check_req_cols(self._rev_summary, 'res_gids')
```

Comment:
Strings (i.e. 'res_gids') could be class- or module-level constants for flexibility/less refactoring errors in the future

Reply:
I like this! It was a kwarg before but wasnt even exposed because it never changes in the sc table. Class attr is a nice in between.